platform/src/main/java/org/stellar/anchor/platform/config/PropertyClientsConfig.java

```diff
@@ -37,82 +33,85 @@ public boolean supports(@NotNull Class<?> clazz) {
 
   @Override
   public void validate(@NotNull Object target, @NotNull Errors errors) {
```

Comment:
Unit tests should be implemented at `PropertyClientsConfigTests`.

Reply:
It was tested in file `ClientsConfigTest`

Reply:
`ClientsConfigTest` should be renamed as `PropertyClientsConfigTests` for JUnit naming convention. 
Reference: https://www.baeldung.com/java-unit-testing-best-practices

Reply:
I've been trying to follow the naming convention and keep it unified in the codebase. I don't quite get what's going wrong with test under the name of ClientsConfigTests?